python/src/aiconfig/editor/server/server.py

```diff
@@ -335,7 +335,8 @@ def run() -> FlaskResponse:
 
     # Allow user to modify their environment keys without reloading the server.
     # Execution time of `0.001s` is arbitrary, but should be small enough to not be noticeable.
-    dotenv.load_dotenv()
+    # Should override be set to True?
+    dotenv.load_dotenv(state.env_file_path, override= True)
```

Comment:
Let's do `override=(state.env_file_path is not None)` (or whatever the clean python way is). We should not override if no path is set since that changes behaviour

Reply:
thanks, +1. You're pretty good at python 😉